src/averageAges.js

```diff
@@ -14,12 +30,13 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const AverageManAge = century
```

Comment:
It's not avarageManAge, rather menAgesSum
```suggestion
  const menAgesSum = century
```